### PR TITLE
Daniels fixes

### PR DESCRIPTION
**Pull request by dtbielejeski**

### CODE REVIEW

File: ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java [hunk 1]
```diff
@@ -69,18 +65,15 @@ public void initialize(URL location, ResourceBundle resources) {
...
-                // Clear all of the previous text-fields
-                clearTextField();
-                // Initialize all of the text-fields
-                title.setText(getTitle());
-                firstName.setText(getFirstName());
-                lastName.setText(getLastName());
-                prevCourseLoad.setText(getPrevCourseLoad());
-                currCourseLoad.setText(getCurrCourseLoad());
-            }
+        facultyID.valueProperty().addListener((observable, oldValue, newValue) -> {
```
Comment (xion6136) [inline ClassySchedule/src/main/java/users/DeleteFacultyFromDatabaseController.java:70]: Does this work the same way as before?

**Review by xion6136 — COMMENTED** [review]
Had a question. Other than that everything else looks fine. I don't know about the fxml files, but it should be good if you got it via scenebuilder.

**Review by elizabethwhalen — APPROVED** [review]
Yay for new fxmls!

**Review by xion6136 — APPROVED** [review]
Good job! Thanks for the carry!

**Pull request merged**